packages/neon/neon/lib/src/pages/home.dart

```diff
@@ -96,6 +96,8 @@ class _HomePageState extends State<HomePage> {
       }
     });
 
+    registerGlobalPopups(context);
```

Comment:
That's bad :thinking:

Reply:
Ok we could add our own [ScaffoldMessenger](https://api.flutter.dev/flutter/material/ScaffoldMessenger-class.html).

Or we could rely on the one provided to us by MaterialApp (didn't know it will do so).
I'll probably have a look at it tomorrow,

I always thought that we need to run this inside a scaffold but that reminds me of the Scaffold.of(context) to ScaffoldMessenger.of(context) migration from a year or so ago. Makes more sense now.

Reply:
I still thought about it yesterday and this wouldn't really work.
While we have a parent ScaffoldMessenger the messenger itself wouldn't have a scaffold descendant yet thus failing.

Maybe just adding a delay. but that`s ugly.